src/app/app.component.ts

```diff
@@ -1,13 +1,96 @@
-import { Component, ViewEncapsulation } from '@angular/core';
-
+import { Component } from '@angular/core';
+export class Hero {
+  id: number;
+  name: string;
+}
+const HEROES: Hero[] = [
+  { id: 11, name: 'Mr. Nice' },
+  { id: 12, name: 'Narco' },
+  { id: 13, name: 'Bombasto' },
+  { id: 14, name: 'Celeritas' },
+  { id: 15, name: 'Magneta' },
+  { id: 16, name: 'RubberMan' },
+  { id: 17, name: 'Dynama' },
+  { id: 18, name: 'Dr IQ' },
+  { id: 19, name: 'Magma' },
+  { id: 20, name: 'Tornado' }
+];
 @Component({
-  selector: 'toh-app',
-  encapsulation: ViewEncapsulation.None,
-  styles: [require('./app.component.scss')],
```

Comment:
As mentioned below, you'll probably be wise to stick to external file for CSS. Especially if you want to use Sass, as we've done here. But keep following the tutorial for now. We'll do this part later.